test/test_stand_by_cost.py

```diff
@@ -2,13 +2,15 @@
 import numpy as np
 import pandas as pd
 import pytest
-from conftest import SUPPORTED_APIS, optimize
+from conftest import optimize
 from numpy.testing import assert_array_almost_equal as equal
 
 import pypsa
 
+SUPPORTED_APIS_STAND_BY_COST = ["linopy", "native"]
 
-@pytest.mark.parametrize("api", SUPPORTED_APIS)
+
+@pytest.mark.parametrize("api", SUPPORTED_APIS_STAND_BY_COST)
 def test_stand_by_cost(api):
```

Comment:
Hello and thanks for taking care of the PR :)
The functionality seems ok to me.

As little comment, I'd revise the exclusion of pyomo as follows.
Rather then creating a new support app variable, in this test function, I'd add an if close like:

If api=="pyomo":
    Return

The effect should be the same but I think it is less impactful.

Thanks for taking care of it :D